core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.html

```diff
@@ -274,7 +273,7 @@
     width: 100%;
   }
   .oppia-active-opportunities-tab a {
-    color: #009688;
```

Comment:
Original color:
![Screen Shot 2021-04-09 at 3 47 13 PM](https://user-images.githubusercontent.com/35385366/114247840-01662380-994b-11eb-8aba-7849d002cd22.png)

Reply:
@mschanteltc So the issue in this case is that using #9688 for the foreground color and white as the background color fails our accessibility audits. 
You can see this in the wave plugin
<img width="344" alt="Screen Shot 2021-04-14 at 9 20 51 AM" src="https://user-images.githubusercontent.com/35385366/114744903-05a38f80-9d03-11eb-931d-5aad79227326.png">

Reply:
Ahh got it, so I would assume that #00645C should pass, correct?

Reply:
Yes it would, you can use this website to check if the colors meet the accessibility standards https://webaim.org/resources/contrastchecker/

Reply:
This could be helpful for future color palettes :)

Reply:
Should we be using #00645C here instead of #00645D then?

Reply:
@sagangwee Chantel had new color reccomendations via this comment https://github.com/oppia/oppia/pull/12457#issuecomment-819930887